src/edge/importer.py

```diff
@@ -122,9 +141,12 @@ def annotate(self, fragment):
                                                          .filter(fragment=fragment)}
 
         for feature in self.__features:
+            t0 = time.time()
             f_start, f_end, f_name, f_type, f_strand, f_qualifiers = feature
             # print('  %s %s: %s-%s %s' % (f_type, f_name, f_start, f_end, f_strand))
             self._annotate_feature(fragment, f_start, f_end, f_name, f_type, f_strand, f_qualifiers)
+            print('annotate feature: %.4f\r' % (time.time() - t0,), end="")
```

Comment:
Ideally, we should convert all these print statements into logger but let's leave that in the future refactoring.